.github/workflows/canister-tests.yml

```diff
@@ -482,6 +482,10 @@ jobs:
       # Make sure that one failing test does not cancel all other matrix jobs
       fail-fast: false
 
+    env:
+      # Suffix used for tagging artifacts
+      artifact_suffix: ${{ contains(matrix.domain, 'internetcomputer') && 'current' || 'legacy' }}-${{ matrix.device }}-${{ matrix.shard }}
```

Comment:
Given the domain migration has been abandoned I'm not sure `current` and `legacy` are really that clear. Why not simply have the domain be part of the `artifact_suffix`?

Reply:
because it's used as an artifact name, which is quite picky about allowed characters IIRC; `current` and `legacy` were also not too long, plus we use this pretty much everywhere else so I thought it was appropriate